projects/ion/src/lib/radio/radio.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container">
```

Comment:
Como na maioria dos componentes tem a classe `container`, seria bom especificar essa aqui pra evitar problemas com o `/deep/ .container {...}`.

```suggestion
<div class="ion-input-radio">
```